### DISCUSSION TITLE
How to compile locally?

### DISCUSSION THREAD

**Discussion by msxfXF** [Kiln-AI/Kiln#81] [Q&A]
Under Linux, use the compiled binary operation

error message: "libm.so.6: version `glibc_2.38 'not found"
So I hope to complete the compilation locally, Thanks

**Comment by scosman** [comment]
Are you trying to compile or trying to run the release binary?

looks like you need to install or reinstall gcc

this file shows the full build process, but it requires a few tools (npm, up, python) so a binary is easier by far: https://github.com/Kiln-AI/Kiln/blob/main/.github/workflows/build_desktop.yml
